src/main.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+#[cfg(target_arch = "x86_64")]
```

Comment:
Maybe link to an issue tracking this. We'll need backtrace eventually.

Reply:
I'll add an issue.

Reply:
https://github.com/firecracker-microvm/firecracker/issues/955